doc/releases/changelog-0.25.0.md

```diff
@@ -20,84 +20,72 @@
+  - [First quantization](https://en.wikipedia.org/wiki/First_quantization) using a plane-wave basis via the `FirstQuantization` class:
 
-  ```python
-  import pennylane as qml
-  from pennylane import numpy as np
-  
-  n = 100000        # number of plane waves
-  eta = 156         # number of electrons
-  omega = 1145.166  # unit cell volume in atomic units
-  
-  algo = FirstQuantization(n, eta, omega)
-  ```
-  
-  ```pycon
-  >>> print(algo.gates, algo.qubits)
-  1.10e+13, 4416
-  ```
+    ```pycon
+    >>> n = 100000        # number of plane waves
+    >>> eta = 156         # number of electrons
+    >>> omega = 1145.166  # unit cell volume in atomic units
+    >>> algo = FirstQuantization(n, eta, omega)
+    >>> print(algo.gates, algo.qubits)
+    1.10e+13, 4416
+    ```
 
   - [Second quantization](https://en.wikipedia.org/wiki/Second_quantization) with a double-factorized Hamiltonian via the 
-  [`DoubleFactorization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.DoubleFactorization.html) class: 
+    `DoubleFactorization` class: 
  
-  ```python
-  import pennylane as qml
-  from pennylane import numpy as np
-  
-  symbols  = ['O', 'H', 'H']
-  geometry = np.array([[0.00000000,  0.00000000,  0.28377432],
-                       [0.00000000,  1.45278171, -1.00662237],
-                       [0.00000000, -1.45278171, -1.00662237]], requires_grad = False)
-  
-  mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
-  core, one, two = qml.qchem.electron_integrals(mol)()
-  
-  algo = DoubleFactorization(one, two)
-  ```
-
-  ```pycon
-  >>> print(algo.gates, algo.qubits)
-  103969925, 290
-  ```
-  
-  The methods of the [`FirstQuantization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.FirstQuantization.html) and the [`DoubleFactorization`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.resource.DoubleFactorization.html) classes 
+    ```python
+    symbols = ["O", "H", "H"]
+    geometry = np.array(
+        [
+            [0.00000000, 0.00000000, 0.28377432],
+            [0.00000000, 1.45278171, -1.00662237],
+            [0.00000000, -1.45278171, -1.00662237],
+        ],
+        requires_grad=False,
+    )
+
+    mol = qml.qchem.Molecule(symbols, geometry, basis_name="sto-3g")
+    core, one, two = qml.qchem.electron_integrals(mol)()
+
+    algo = DoubleFactorization(one, two)
+    ```
+
+    ```pycon
+    >>> print(algo.gates, algo.qubits)
+    103969925, 290
+    ```
+    
+  The methods of the `FirstQuantization` and the `DoubleFactorization` classes 
   can be also accessed individually without instantiating an instance of the class:
 
-  - The number of logical qubits with `qubit_cost`
-  - The number of non-Clifford gates with `gate_cost`
+  + The number of logical qubits with `qubit_cost`
+  + The number of non-Clifford gates with `gate_cost`
```

Comment:
This is very minor, but since there is only a single codeblock, you could unindent the codeblock and remove the bullet points. E.g.,

````markdown
The methods of the `FirstQuantization` and the `DoubleFactorization` classes,
such as `qubit_cost` (number of logical qubits) and `gate_cost` (number
of non-Clifford gates), can be also accessed as static methods:

```pycon
>>> qml.resource.FirstQuantization.qubit_cost(100000, 156, 169.69608, 0.01)
4377
>>> qml.resource.FirstQuantization.gate_cost(100000, 156, 169.69608, 0.01)
3676557345574
```
````